### PR TITLE
first submit of js2 week3 homework

### PR DESCRIPTION
**Pull request by shanhaider23**
I try to follow the checklist and completed all steps.   
 [x]-Does every file run without errors and with the correct results?
 [x]- Have you used const and let and avoided var?
 [x]-Do the variable, function, and argument names you created to follow the Naming Conventions?
 [x]- Is your code well-formatted (see Code Formatting)?

### CODE REVIEW

File: javascript/javascript2/week3/main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    document.getElementById("countL").innerText = playerL;
+  } else if (e.key == "s") {
+    playerS++;
+    document.getElementById("countS").innerText = playerS;
+  }
+};
+
+const inputTime = function () {
+  return document.getElementById("inputnumber").value;
+};
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/main.js]: Every time `inputTime` is called `document.getElementById("inputnumber")` has to search the DOM for this element. You can save it in a variable once to save lots of calls to `getElementById`.
```suggestion
const inputNumberElement = document.getElementById("inputnumber");
const inputTime = function () {
  return inputNumberElement.value;
};
```

File: javascript/javascript2/week3/variableFunction.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const array = [
+  (firstFunction = function () {
+    console.log("first function");
+  }),
+  (secondFunction = function () {
+    console.log("second function");
+  }),
+  (thirdFunction = function () {
+    console.log("third function");
+  }),
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/variableFunction.js]: The variables are never used so you can remove them.
```suggestion
  (function () {
    console.log("first function");
  }),
  (function () {
    console.log("second function");
  }),
  (function () {
    console.log("third function");
  }),
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week3/main.js (hunk 1 above)
Comment (shanhaider23) [reply javascript/javascript2/week3/main.js]: Thanks yes it is a good suggestion

File: javascript/javascript2/week3/variableFunction.js (hunk 2 above)
Comment (shanhaider23) [reply javascript/javascript2/week3/variableFunction.js]: thanks, I will change it

**Pull request merged**